metricbeat/module/http/server/config.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 type HttpServerConfig struct {
-	Paths       []PathConfig `config:"paths"`
-	DefaultPath PathConfig   `config:"default_path"`
+	Paths       []PathConfig `config:"server.paths"`
+	DefaultPath PathConfig   `config:"server.default_path"`
```

Comment:
LGTM, if you want to do a softer deprecation you could still read old settings before these, but I don't think this is a hard requirement.